### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by alejolale**
I made the home page, presentation  name of game, start button, planet image.

### CODE REVIEW

**Comment by superfrancisco2019** [comment]
review made, everything ok

**Comment by Aebrathia** [comment]
Hi,

- Components folder should be lowercase
- Each component and their folder should be written in PascalCase
- CSS Stylesheets should be linked at the end of head element
- Use import instead of require and outside of component
- Be careful of spacing and indentation
- You don't need 3 elements to style your start button
- Make your text uppercase using styles
- Indent everything inside media
- style generic html elements in index.css not in components
- You don't need @media min width 320px

Thanks !

File: public/index.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <!-- Link for google fonts - name of the game in home-page component-->
+    <link href="https://fonts.googleapis.com/css?family=Orbitron&display=swap" rel="stylesheet">
```
Comment (Aebrathia) [inline public/index.html]: Move this

**Review by superfrancisco2019 — APPROVED** [review]
all changes have been made after the first pull request

**Comment by Aebrathia** [comment]
You should name your component folder and files with PascalCase : HomePage/HomePage.js

**Review by superfrancisco2019 — APPROVED** [review]
ok

**Comment by Aebrathia** [comment]
Images should be placed in the folder of the component hey are in :
- `/src/components/HomePage/HomePage.js`
- `/src/components/HomePage/HomePage.css`
- `/src/components/HomePage/my-image.png`

Images should also have explicit kebab-case names.

In case the same image is used by multiple components you can create a folder like this :
- `/src/images/` or
- `/src/img/`

If you don't need an image anymore remove it from the project.

**Review by Aebrathia — CHANGES_REQUESTED** [review]
Images should be placed in the folder of the component hey are in :

    /src/components/HomePage/HomePage.js
    /src/components/HomePage/HomePage.css
    /src/components/HomePage/my-image.png

Images should also have explicit kebab-case names.

In case the same image is used by multiple components you can create a folder like this :

    /src/images/ or
    /src/img/

If you don't need an image anymore remove it from the project.

**Comment by superfrancisco2019** [comment]
ok

**Comment by Aebrathia** [comment]
Congrats first PR of all projects to be merged!

**Review by Aebrathia — APPROVED** [review]

**Pull request merged**